src/platform/user/profile/vet360/tests/actions/transactions.unit.spec.js

```diff
@@ -133,7 +45,7 @@ describe('validateAddress', () => {
           addressLine1: '400 N 65th St',
           addressType: 'DOMESTIC',
           city: 'Seattle',
-          countryName: 'USA',
+          countryName: 'United States',
```

Comment:
If it isn't already, this is definitely a tricky one that makes a good candidate for a constants file

Reply:
🤔  How do you mean? This one is a bit convoluted, but this test is really testing the data that comes back from the new validation API service, but in this test we were using mock data. I updated the mock data so that instead of sending back `USA`  it was sending back `United States`, which is what the real service returns. So making that change required updating this test.

Reply:
There is probably a better/less brittle way to test this, but since this was an existing test and I'm in a mad dash, I wanted to just fix it as quickly as possible 😞

Reply:
Ah nvm! I was thinking the value representing USA in the payload for what we send back to Vet360